engine/input/keyboard.go

```diff
@@ -36,14 +41,22 @@ func keyCallback(
 	}
 
 	if action == glfw.Press {
+		if key == glfw.KeyU {
+			timer = time.Now()
+		}
 		if key == glfw.KeyEscape {
 			w.SetShouldClose(true)
 
 		}
 		lastKeyPressed = key
 		KeysPressedDown[key] = true
 		KeysPressed[key] = true
+
 	} else if action == glfw.Repeat {
+		if key == glfw.KeyU {
+			newTime := time.Since(timer).Milliseconds()
+			fmt.Println(newTime)
```

Comment:
Should probably not print numbers without context to `stdout`

Reply:
this is debug didn't delete before pull requesting